.github/workflows/build.yml

```diff
@@ -1,25 +1,39 @@
 name: .NET Core CI
 
-on: [push, pull_request]
+on:
+  push:
+    paths-ignore:
+      - '.github/*'
+      - '.github/*_TEMPLATE/**'
+      - '*.md'
+  pull_request:
+    paths-ignore:
+      - '.github/*'
```

Comment:
Surely we should be running the CI workflow when somebody makes changes to the CI workflow?

Reply:
It only ignores files in the first subdirectory. It's not recursive (i.e. `.github/workflows`)